Modules/ITS/src/ITSClusterCheck.cxx

```diff
@@ -163,6 +175,44 @@ void ITSClusterCheck::beautify(std::shared_ptr<MonitorObject> mo, Quality checkR
       h->GetListOfFunctions()->Add(tShifterInfo->Clone());
   }
 
+  if (mo->getName().find("EmptyLaneFractionGlobal") != std::string::npos) {
+    auto* h = dynamic_cast<TH1D*>(mo->getObject());
+    if (checkResult == Quality::Good) {
+      status = "Quality::GOOD";
+      textColor = kGreen;
+      positionX = 0.05;
+      positionY = 0.91;
+    } else if (checkResult == Quality::Bad) {
+      status = "Quality::BAD (call expert)";
+      textColor = kRed;
+      if (strcmp(checkResult.getMetadata("EmptyLaneFractionGlobal").c_str(), "bad") == 0) {
+        MaxEmptyLaneFraction = o2::quality_control_modules::common::getFromConfig<float>(mCustomParameters, "MaxEmptyLaneFraction", MaxEmptyLaneFraction);
+        tInfoSummary = std::make_shared<TLatex>(0.12, 0.5, Form(">%.0f %% of the lanes are empty", MaxEmptyLaneFraction * 100));
+        tInfoSummary->SetTextColor(kRed);
+        tInfoSummary->SetTextSize(0.05);
+        tInfoSummary->SetTextFont(43);
+        tInfoSummary->SetNDC();
+        h->GetListOfFunctions()->Add(tInfoSummary->Clone());
+        tInfo = std::make_shared<TLatex>(0.1, 0.11, Form("#bf{%s}", "Threshold value"));
+        tInfo->SetTextColor(kRed);
+        tInfo->SetTextSize(0.05);
+        tInfo->SetTextFont(43);
+        h->GetListOfFunctions()->Add(tInfo->Clone());
+        tInfoLine = std::make_shared<TLine>(0, 0.1, 4, 0.1);
+        tInfoLine->SetLineColor(kRed);
+        tInfoLine->SetLineStyle(9);
+        h->GetListOfFunctions()->Add(tInfoLine->Clone());
```

Comment:
Do I understand correctly that you add TLIne and "threshold value" text only when this histo is BAD? Can you plot this info all the time?